transactor/src/main/scala/io/mediachain/copycat/TransactorService.scala

```diff
@@ -440,22 +441,22 @@ object TransactorService {
   }
   
   def journalBlockReferenceToEvent(ref: Reference): JournalEvent = {
-    JournalEvent().withEvent(Event.JournalBlockEvent(refToRPCMultihashRef(ref)))
+    JournalEvent(JournalEvent.Event.JournalBlockEvent(refToRPCMultihashRef(ref)))
```

Comment:
Yeah, the java protobuf interface is not the prettiest thing; the naming as JournalEvent is intentional as it actually shadows the state machine events in protobuf-land